### PR TITLE
3729: fixing handling of stacked fluid container by tank

### LINKED ISSUE #3729: Cannot fill up stacked fluid containers
1.11.2 - BC 7.99.6
I cannot fill up an empty IC2 universal capsule by right-clicking on a BC tank full of lava

### PR DESCRIPTION
**Pull request by seriar**
Adding a check for quantity of items. If there are more than one item try to use one item from the stack. If item from the stack is filled `give it to the player or drop it at their feet` as is done in core forge.

Closes #3729

### CODE REVIEW

File: common/buildcraft/factory/tile/TileTank.java [hunk 1]
```diff
@@ -43,6 +43,7 @@
 import buildcraft.lib.misc.data.IdAllocator;
 import buildcraft.lib.net.PacketBufferBC;
 import buildcraft.lib.tile.TileBC_Neptune;
+import net.minecraftforge.items.ItemHandlerHelper;
```
Comment (AlexIIL) [inline common/buildcraft/factory/tile/TileTank.java]: Can you move this up to just after  ````import net.minecraftforge.fml.relauncher.SideOnly;```` (That will fix checkstyle)

**Comment by AlexIIL** [comment]
Thanks!

**Comment by AEnterprise** [comment]
Thanks for contributing to BuildCraft, if you are on the discord server (or join in the future) please post your discord username and discriminator here to get the contributors role

**Comment by seriar** [comment]
seriar#6217

**Pull request merged**